CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+This document describes how to go about contributing to the ISOBlue-Avena repository as well as defining how we manage pull requests, releases, and other collaborative features
+
+
+## General Guidelines
+If you are looking to help us, please visit our [issue tracker](https://github.com/OATS-Group/isoblue-avena/issues). If you would like to suggest or discuss a feature/bug, please email our mailing list found here https://groups.google.com/forum/?oldui=1#!forum/isoblue. Please ensure you mention the 'Avena' version of ISOBlue.
```

Comment:
> https://groups.google.com/forum/?oldui=1#!forum/isoblue

Is there a better way to link to our mailing group? URL seems a but awkward

Reply:
How about: _"If you would like to suggest or discuss a feature or bug, please email our "ISOBlue" Google Groups mailing list at: $URL"_.